Opio-Emmanuel-Omona/fast-food-fast-8.1
# #164346537 User can signup and login

#### What does this PR do?
This PR enables users to signup and sign in to the application.

#### Description of Task to be completed?
* Create login page
* Create signup page
* Write tests

#### How should this be manually tested?
* Visit this [URL.](https://fast-food-fast-staging-pr-7.herokuapp.com/)
* Use the screenshots as expected results.
* Create an account and sign in.

#### What are the relevant pivotal tracker stories?
[ #164346537](https://www.pivotaltracker.com/story/show/164346537)

#### Screenshots
![image](https://user-images.githubusercontent.com/26769886/53674776-6695b800-3ca1-11e9-9c2d-e189ca1651a3.png)

![image](https://user-images.githubusercontent.com/26769886/53674778-6ac1d580-3ca1-11e9-9765-6dcee378ab6a.png)
